wasm/src/programs/data/ciphertext.rs

```diff
@@ -57,6 +57,14 @@ impl Ciphertext {
 
     /// Decrypt a ciphertext using the view key of the transition signer, transition public key, and
     /// (program, function, index) tuple.
+    ///
+    /// @param {ViewKey} view_key The view key of the transition signer.
+    /// @param {Group} transition_public_key The transition public key used to encrypt the ciphertext.
+    /// @param {string} program The program ID of the function that encrypted the ciphertext.
+    /// @param {string} function_name The function name that encrypted the ciphertext.
+    /// @param {u16} index The index of the input parameter that was encrypted.
```

Comment:
```suggestion
    /// @param {u16} index The index of the input or output parameter that was encrypted.
```

This applies to inputs AND outputs.